src/tests/data_handlers/user_data_handler.mock.test.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+jest.mock('axios');
+import UserDataHandler from '../../data_handlers/user_data_handler';
+import testData from '../../../data/users.json';
+import serchParamsTestData from '../../../data/isMatchingAllSearchParamsTestData.json';
+import findUsersTestData from '../../../data/findUsersTestData.json';
+describe('loadUsers() method check', () => {
+  let userDataHadlerObject;
+  beforeEach(() => {
+    userDataHadlerObject = new UserDataHandler();
+  });
+  test('loadUsers() returns expected error if axios call failed', async () => {
```

Comment:
It's better to use manual test case summary approach. For ex. "item" should return "specific result", while/if "conditions"